.github/workflows/workflow.yaml

```diff
@@ -256,12 +256,16 @@ jobs:
           tar zxvf trivy_${{ env.TRIVY_VERSION }}_Linux-64bit.tar.gz
           echo "$(pwd)" >> $GITHUB_PATH
         env:
-          TRIVY_VERSION: "0.22.0"
+          TRIVY_VERSION: "0.29.2"
+      
+      - name: Run trivy on git repository
+        run: |
+          trivy fs --format table --ignore-unfixed --skip-dirs website --security-checks vuln .
 
       - name: Build docker images
         run: make docker-build
 
-      - name: Run trivy
+      - name: Run trivy on images
         run: |
           for img in "openpolicyagent/gatekeeper:latest" "openpolicyagent/gatekeeper-crds:latest"; do
             for vuln_type in "os" "library"; do
```

Comment:
looks like we'll have to remove exit code from here too unless existing vulns are fixed. thoughts @ritazh?

Reply:
yea let's remove exit code for now until we have a better process.

Reply:
Done.